### ISSUE TITLE
Small documentation error on wiki/Compiling-on-Debian

### ISSUE THREAD

**Issue by lanroth** [strawberrymusicplayer/strawberry#690]
TL;DR: add `libtag1-dev` to `apt-get` command on [wiki/Compiling-on-Debian](https://github.com/strawberrymusicplayer/strawberry/wiki/Compiling-on-Debian)

**Describe the bug**

On a fresh version of Debian 10 (Raspbian) [these instructions](https://github.com/strawberrymusicplayer/strawberry/wiki/Compiling-on-Debian) produced `No package 'taglib' found`.

After executing `apt-get install libtag1-dev` I could successfully compile.

**Expected behavior**

A successful build.

**Screenshots:**

```
-- Checking for module 'taglib>=1.11.1'
--   No package 'taglib' found
```

**System Information:**
 - Operating system:

```
pi@raspberrypi:~ $ lsb_release -a
No LSB modules are available.
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 10 (buster)
Release:        10
Codename:       buster
```

 - Strawberry Version:

latest from master branch

Love the project - thanks for your hard work :-)

**Comment by jonaski** [comment]
Done
